cmd/zpool/zpool_util.h

```diff
@@ -125,6 +125,12 @@ vdev_cmd_data_list_t *all_pools_for_each_vdev_run(int argc, char **argv,
 
 void free_vdev_cmd_data_list(vdev_cmd_data_list_t *vcdl);
 
+int check_device(const char *path, boolean_t force,
+    boolean_t isspare, boolean_t iswholedisk);
+boolean_t check_sector_size_database(char *path, int *sector_size);
+void vdev_error(const char *fmt, ...);
+int check_file(const char *file, boolean_t force, boolean_t isspare);
```

Comment:
After taking another look at this, I think a nicer approach would be to move this functionality in to `libzutil`.  Adding `zfs_isnumber()` to `lib/libzutil/zutil_nicenum.c` would make sense and be generally useful.  Plus, `lib/libzutil/zutil_import.c` alreadys contain platform specific code and would be a natural home for `check_device()`, `check_file()`, `check_sector_size_database()` after the refactoring in #9356 is settled.

This would have the advantage of keeping the `zpool` command itself largely portable with the platform specific being contained in the library.

Reply:
The vdev checks were difficult to move to libzutil when I tried. It introduces a circular dependency with libzfs. I've left them where they are for now.

Reply:
OK, thanks for investigating.  Then this LGTM.